packages/examples/config/tests.js

```diff
@@ -0,0 +1,15 @@
+/** Jest test setup file. */
```

Comment:
The test config was added because I realized the ComponentExamples snapshot test will need to move into this package.